carbon-design-system/carbon
# fix(a11y): fixing Form Components in Windows High Contrast Mode

Closes #6882 

fixing Form Components in Windows High Contrast Mode

#### Changelog

**Changed**

Button component
Number Input component
Toggle component

#### Testing / Reviewing

On a Windows 10 machine, go to `settings --> ease of access --> high contrast` and flick the switch. Compare the storybook in Edge (Should be in High Contrast) to Chrome (Should look normal). 
For **Firefox**, you'll need to open it up and type `about:config` into the address bar. Then, search for `layout.css.prefers-contrast` and set it to `true`.

- Visit [Form](http://192.168.1.158:9000/?path=/story/form--default) component
- Verify that the number input shows a bounding box and has a visible focus state
- Verify that the toggles are visually accessible
- Verify that the radio buttons are visually accessible
- Verify that the iconography in the search bar is visually accessible
- Verify that the Select element has a focus state
- Verify that the Submit button has a focus state
